main.sh

```diff
@@ -32,8 +32,8 @@ if [ "${BASH_SOURCE[0]}" != "${0}" ]; then
     __b3bp_tmp_source_idx=1
   fi
 else
-  [ ! -z "${__usage+x}" ] && unset __usage
-  [ ! -z "${__helptext+x}" ] && unset __helptext
+  [ ! -z "${__usage+x}" ] && unset -v __usage
+  [ ! -z "${__helptext+x}" ] && unset -v __helptext
```

Comment:
It seems that you could even get rid of the unary operator altogether.  Eventually we're going to transition to use only `[[ ... ]]` but we can do this later since this PR is blocking your other PR (effectively) I'm going to merge and we we can tweak later as needed/wanted.